lib/internal/bootstrap/cache.js

```diff
@@ -42,6 +43,15 @@ const cannotUseCache = [
   'internal/bootstrap/node'
 ].concat(depsModule);
 
+if (process.config.variables.v8_enable_inspector !== 1) {
```

Comment:
May be worthwhile including a quick comment with these that says *why* they cannot use the cache, just so it's clear to folks coming along later.